frontend/src/static/css/style.scss

```diff
@@ -195,8 +199,7 @@ header {
 
     .tab-content {
       display: flex;
-      padding: 1rem 1.5rem;
-      padding-top: 4rem;
+
```

Comment:
Why extra line?

Reply:
Ahhh didn't notice this. Will make the necessary change and push.